playwright/_impl/_helper.py

```diff
@@ -239,9 +242,12 @@ def impl() -> None:
         else:
             impl()
 
+        [handled, *_] = await asyncio.gather(handled_future, *handler_coros)
```

Comment:
what does the *_ do?

Reply:
Gives us JS-like semantics!

It unpacks zero or more. Without, `[a] = [0, 1]` and `[a, _] = [0]` would go 💥